tools/c7n_azure/tests_azure/templates/locked.json

```diff
@@ -25,16 +23,47 @@
             },
             "dependsOn": []
         },
-        {	
-            "name": "cctestlockfilter",	
-            "type": "Microsoft.Authorization/locks",	
-            "apiVersion": "2016-09-01",	
-            "dependsOn": [	
-                "[resourceId('Microsoft.Compute/disks', parameters('diskName'))]"	
-            ],	
-            "properties": {	
-                "level": "CanNotDelete",	
-                "notes": "This lock is for the cc test_filter_resource_lock test"	
+        {
+            "name": "cctestlockfilter",
+            "type": "Microsoft.Authorization/locks",
+            "apiVersion": "2016-09-01",
+            "dependsOn": [
+                "[resourceId('Microsoft.Compute/disks', variables('diskName'))]"
+            ],
+            "properties": {
+                "level": "CanNotDelete",
+                "notes": "This lock is for the cc test_filter_resource_lock test"
+            }
+        },
+        {
+            "apiVersion": "2015-05-01-preview",
+            "type": "Microsoft.Sql/servers",
+            "location": "EastUS2",
+            "name": "[variables('servers_cctestsqlserver_name')]",
+            "properties": {
+                "administratorLogin": "custodian",
+                "administratorLoginPassword": "Cust0dianPassw0rd",
```

Comment:
Is it possible for us to deploy the server without hard coding the password? 
Maybe we pass it as a parameter that we randomly generate from the provision script? 
I think this may result in a flag from the security audits.